shuup/simple_cms/templates/shuup/simple_cms/custom_sidebar.jinja

```diff
@@ -0,0 +1,44 @@
+{% extends "shuup/front/base.jinja" %}
+
+{% block title %}{{ page.title }}{% endblock %}
+{% block content_title %}
+    {% if page.render_title %}
+        {{ page.title }}
+    {% endif %}
+    <div class="col-md-3 categories-nav" style="font-size: 15px">
+        {{ category_macros.render_category_menu() }}
+    </div>
+{% endblock %}
+{% block extrameta %}
+{{ macros.render_metadata(page, page.title, page.get_html(), "page") }}
+{% endblock %}
+
+{% block breadcrumb %}
+    <ol class="breadcrumb">
+        <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
+        {% for ancestor in page.get_ancestors() if ancestor.is_visible() %}
+            <li><a href="/{{ ancestor.url }}">{{ ancestor.title }}</a></li>
+        {% endfor %}
+        <li class="active">{{ page.title }}</li>
+    </ol>
+{% endblock %}
+{% block content %}
+
```

Comment:
this template should looks like this:
```jinja
{% extends "shuup/front/base.jinja" %}

{% block title %}{{ page.title }}{% endblock %}
{% block content_title %}
    {% if page.render_title %}
        {{ page.title }}
    {% endif %}
{% endblock %}
{% block extrameta %}
{{ macros.render_metadata(page, page.title, page.get_html(), "page") }}
{% endblock %}

{% block breadcrumb %}
    <ol class="breadcrumb">
        <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
        {% for ancestor in page.get_ancestors() if ancestor.is_visible() %}
            <li><a href="/{{ ancestor.url }}">{{ ancestor.title }}</a></li>
        {% endfor %}
        <li class="active">{{ page.title }}</li>
    </ol>
{% endblock %}
{% block content %}
    <div class="col-md-3 categories-nav">
        {{ category_macros.render_category_menu() }}
    </div>
    <div class="col-md-8">
        {{ page.get_html()|safe }}
        {% if page.list_children_on_page %}
            {# TODO: add pagination for children list #}
            {% for child in page.children.visible(request.shop).order_by("-available_from") %}
                <div class="row">
                    <a href="/{{ child.url }}"><h2>{{ child.title }}</h2></a>
                    {% if child.available_from and page.show_child_timestamps %}
                        <small>{{ child.available_from|datetime }}</small>
                    {% endif %}
                    <p>{{ child.get_html()|safe|striptags|truncate(256) }}</p>
                </div>
            {% endfor %}
        {% endif %}
        {% placeholder "cms_page" %}{% endplaceholder %}
    </div>
{% endblock %}
{% block extrajs %}
    {{ category_macros.render_extrajs() }}
{% endblock %}
```

Reply:
The title is `above` the sidebar, also title & sidebar aren't aligned.

![screenshot from 2018-08-31 23-48-37](https://user-images.githubusercontent.com/40273438/44937200-6a8b6080-ad78-11e8-94a0-244ac68a0302.png)

Reply:
your solution doesn't work either, the sidebar styles are completely broken, you shouldn't have to apply element inline styles